target/linux/ramips/dts/BARIONET-1000-V1-1-A9.dts

```diff
@@ -0,0 +1,145 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "BARIONET-1000-V1-1-A9", "ralink,mt7620a-soc";
+	model = "Barix Barionet 1000 v1.1.A9";
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+	aliases {
+		spi0 = &spi0;
+		spi1 = &spi1;
+		serial0 = &uartlite;
+		serial1 = &uart;
+	};
+	gpio-leds {
```

Comment:
Your commit message mentiones 11 LED status indicators. Any specific reason why they are not added here?